src/views/tickets/tpp.php

```diff
@@ -3,22 +3,24 @@
  * This template renders the Tribe Commerce ticket form
  *
  * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe-events/tickets/tpp.php
  *
- *     [your-theme]/tribe-events/tickets/tpp.php
+ * @deprecated 4.11.0
```

Comment:
I want to say that this template never gets called anymore and doesn't need any further changes (unless someone disagrees).

Reply:
Don't think it should be deprecated... https://github.com/moderntribe/event-tickets/blob/4.11.4/src/Tribe/Commerce/PayPal/Frontend/Tickets_Form.php#L102

Please confirm.

Reply:
@sc0ttkclark confirmed via Slack DMs that we shouldn't delete this file yet but to create a ticket to investigate its usage: [GTRIA-113]